modules/packages/BLAS.chpl

```diff
@@ -162,6 +159,11 @@ The native BLAS interface can still be accessed by calling routines from the
 .. _AMIN:   https://software.intel.com/en-us/node/50dceaa5-3463-402f-8065-a48bc68e0888
 .. _CABS1:  https://software.intel.com/en-us/node/64961e94-92d0-4671-90e6-86995e259a85
 
+.. BLAS Module TODO:
+  - Clearer compiler errors instead of using where-clauses
+  - Modular implementations using config param-wrapped require statements
+  - More consistent documentation
```

Comment:
I'd add a note somewhere in this top-level documentation (unless I've missed one) that this module is only intended to work with non-distributed dense rectangular arrays.